src/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_6_0/IdnaTest.txt

```diff
@@ -0,0 +1,850 @@
+N;	\u0308\u200D\u0308\u0628B;	[V5 B1 C2];	[V5 B1 C2]
+#T;	a\u0628\u0308\u200D\u0308;	[B5 B6 C2];	[B5 B6]	# GETASCII DOES NOT FAIL ON WINDOWS 8.1
+N;	a\u0628\u0308\u200D\u0308;	[B5 B6 C2];	[B5 B6 C2]
+#T;	A\u0628\u0308\u200D\u0308;	[B5 B6 C2];	[B5 B6]	# GETASCII DOES NOT FAIL ON WINDOWS 8.1
+N;	A\u0628\u0308\u200D\u0308;	[B5 B6 C2];	[B5 B6 C2]
+#T;	a\u0628\u0308\u200D\u0308\u0628b;	[B5 C2];	[B5]	# GETASCII DOES NOT FAIL ON WINDOWS 8.1
+N;	a\u0628\u0308\u200D\u0308\u0628b;	[B5 C2];	[B5 C2]
+#T;	A\u0628\u0308\u200D\u0308\u0628B;	[B5 C2];	[B5]	# GETASCII DOES NOT FAIL ON WINDOWS 8.1
+N;	A\u0628\u0308\u200D\u0308\u0628B;	[B5 C2];	[B5 C2]
+#T;	A\u0628\u0308\u200D\u0308\u0628b;	[B5 C2];	[B5]	# GETASCII DOES NOT FAIL ON WINDOWS 8.1
+N;	A\u0628\u0308\u200D\u0308\u0628b;	[B5 C2];	[B5 C2]
+B;	1234567890ä1234567890123456789012345678901234567890123456;	;	[A4_2]
+B;	1234567890Ä1234567890123456789012345678901234567890123456;	1234567890ä1234567890123456789012345678901234567890123456;	[A4_2]
+B;	www.eXample.cOm;	www.example.com;	
+B;	www.example.com;	;	
+B;	WWW.EXAMPLE.COM;	www.example.com;	
+B;	Www.example.com;	www.example.com;	
+B;	Bücher.de;	bücher.de;	xn--bcher-kva.de
+B;	bücher.de;	;	xn--bcher-kva.de
+B;	BÜCHER.DE;	bücher.de;	xn--bcher-kva.de
+B;	xn--bcher-kva.de;	bücher.de;	xn--bcher-kva.de
+B;	XN--BCHER-KVA.DE;	bücher.de;	xn--bcher-kva.de
+B;	Xn--Bcher-Kva.de;	bücher.de;	xn--bcher-kva.de
+B;	ÖBB;	öbb;	xn--bb-eka
+B;	öbb;	;	xn--bb-eka
+B;	Öbb;	öbb;	xn--bb-eka
+B;	xn--bb-eka;	öbb;	xn--bb-eka
+B;	XN--BB-EKA;	öbb;	xn--bb-eka
+B;	Xn--Bb-Eka;	öbb;	xn--bb-eka
+#B;	XN--fA-hia.dE;	faß.de;	xn--fa-hia.de	#	GETUNICODE FAILS ON WINDOWS 8.1
+T;	βόλος.com;	;	xn--nxasmq6b.com
+N;	βόλος.com;	;	xn--nxasmm1c.com
+B;	ΒΌΛΟΣ.COM;	βόλοσ.com;	xn--nxasmq6b.com
+B;	βόλοσ.com;	;	xn--nxasmq6b.com
+B;	Βόλοσ.com;	βόλοσ.com;	xn--nxasmq6b.com
+B;	xn--nxasmq6b.com;	βόλοσ.com;	xn--nxasmq6b.com
+B;	XN--NXASMQ6B.COM;	βόλοσ.com;	xn--nxasmq6b.com
+B;	Xn--Nxasmq6b.com;	βόλοσ.com;	xn--nxasmq6b.com
+T;	Βόλος.com;	βόλος.com;	xn--nxasmq6b.com
+N;	Βόλος.com;	βόλος.com;	xn--nxasmm1c.com
+#B;	xn--nxasmm1c.com;	βόλος.com;	xn--nxasmm1c.com	#	GETUNICODE FAILS ON WINDOWS 8.1
+#B;	XN--NXASMM1C.COM;	βόλος.com;	xn--nxasmm1c.com	#	GETUNICODE FAILS ON WINDOWS 8.1
+#B;	Xn--Nxasmm1c.com;	βόλος.com;	xn--nxasmm1c.com	#	GETUNICODE FAILS ON WINDOWS 8.1
+#B;	xn--nxasmm1c;	βόλος;	xn--nxasmm1c	#	GETUNICODE FAILS ON WINDOWS 8.1
+#B;	XN--NXASMM1C;	βόλος;	xn--nxasmm1c	#	GETUNICODE FAILS ON WINDOWS 8.1
+#B;	Xn--Nxasmm1c;	βόλος;	xn--nxasmm1c	#	GETUNICODE FAILS ON WINDOWS 8.1
+T;	βόλος;	;	xn--nxasmq6b
+N;	βόλος;	;	xn--nxasmm1c
+B;	ΒΌΛΟΣ;	βόλοσ;	xn--nxasmq6b
+B;	βόλοσ;	;	xn--nxasmq6b
+B;	Βόλοσ;	βόλοσ;	xn--nxasmq6b
+B;	xn--nxasmq6b;	βόλοσ;	xn--nxasmq6b
+B;	XN--NXASMQ6B;	βόλοσ;	xn--nxasmq6b
+B;	Xn--Nxasmq6b;	βόλοσ;	xn--nxasmq6b
+T;	Βόλος;	βόλος;	xn--nxasmq6b
+N;	Βόλος;	βόλος;	xn--nxasmm1c
+T;	www.ශ\u0DCA\u200Dර\u0DD3.com;	;	www.xn--10cl1a0b.com
```

Comment:
It appears the formatting has changed in this file.  See Line 188+ as examples where tabs appear (at least via github) to have been removed